cheroot/server.py

```diff
@@ -1307,9 +1307,7 @@ def communicate(self):  # noqa: C901  # FIXME
         except errors.NoSSLError:
             self._handle_no_ssl(req)
         except Exception as ex:
-            self.server.error_log(
-                repr(ex), level=logging.ERROR, traceback=True,
-            )
+            self.server.log.exception(repr(ex))
```

Comment:
`logging.exception()` can retrieve the exception from the current context so passing it explicitly is not necessary. But as I said earlier, it's something to do in a separate PR.